plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/messages.properties

```diff
@@ -112,34 +112,34 @@ MessageMediator3CreationTool_desc=Create new Message Mediator
 NavigatorGroupName_MessageInputConnector_3046_incominglinks=incoming links
 NavigatorGroupName_MessageOutputConnector_3047_outgoinglinks=outgoing links
 Sequence4CreationTool_title=Sequence
-Sequence4CreationTool_desc=Create new Sequence
+Sequence4CreationTool_desc=Inserts a reference to a sequence
 NavigatorGroupName_SequenceInputConnector_3049_incominglinks=incoming links
 NavigatorGroupName_SequenceOutputConnector_3050_outgoinglinks=outgoing links
 MessageFormatParser_InvalidInputError=Invalid input at {0}
 EventMediator8CreationTool_title=Event
-EventMediator8CreationTool_desc=Create new Event Mediator
+EventMediator8CreationTool_desc=Sends event notifications to an event source, publishes messages to predefined topics
 NavigatorGroupName_EventMediatorOutputConnector_3053_outgoinglinks=outgoing links
 NavigatorGroupName_EventMediatorInputConnector_3052_incominglinks=incoming links
 EntitlementMediator9CreationTool_title=Entitlement
-EntitlementMediator9CreationTool_desc=Create new Entitlement Mediator
+EntitlementMediator9CreationTool_desc=Evaluates user actions against a XACML policy
 ClassMediator10CreationTool_title=Class
-ClassMediator10CreationTool_desc=Create new Class Mediator
+ClassMediator10CreationTool_desc=Creates and executes a custom mediator
```

Comment:
Isn't it something like "Adding a reference to a custom mediator", since we have "mediator project" to create a custom mediator?